webapp/src/components/SynopForm.vue

```diff
@@ -148,35 +139,30 @@
             <v-list-item v-for="(data_item, index) in result.data_items" :key="index">
-              <!-- Place download and inspect buttons on the right of the file names for tablet and desktop -->
-              <div class="hidden-xs">
-                <div class="file-actions">
-                  <div>
-                    {{ data_item.filename }}
-                  </div>
+              <div class="file-actions">
+                <div>
+                  {{ data_item.filename }}
+                </div>
+                <!-- For wider windows, make buttons wider -->
+                <div class="hidden-md-and-down">
                   <div class="file-actions" v-if="data_item.file_url">
-                    <DownloadButton :fileName="data_item.filename" :fileUrl="data_item.file_url" />
-                    <InspectBufrButton :fileName="data_item.filename" :fileUrl="data_item.file_url" />
+                    <DownloadButton :fileName="data_item.filename" :fileUrl="data_item.file_url" :block="true" />
+                    <InspectBufrButton :fileName="data_item.filename" :fileUrl="data_item.file_url" :block="true" />
                   </div>
                   <div class="file-actions" v-if="data_item.data">
-                    <DownloadButton :fileName="data_item.filename" :data="data_item.data" />
-                    <InspectBufrButton :fileName="data_item.filename" :data="data_item.data" />
+                    <DownloadButton :fileName="data_item.filename" :data="data_item.data" :block="true" />
+                    <InspectBufrButton :fileName="data_item.filename" :data="data_item.data" :block="true" />
                   </div>
                 </div>
-              </div>
-              <!-- Place download and inspect buttons below the file names for mobile -->
-              <div class="hidden-sm-and-up">
-                <div>
-                  <div>
-                    {{ data_item.filename }}
-                  </div>
+                <!-- For narrow windows, make buttons less wide -->
+                <div class="hidden-lg-and-up">
                   <div class="file-actions" v-if="data_item.file_url">
-                    <DownloadButton :fileName="data_item.filename" :fileUrl="data_item.file_url" />
-                    <InspectBufrButton :fileName="data_item.filename" :fileUrl="data_item.file_url" />
+                    <DownloadButton :fileName="data_item.filename" :fileUrl="data_item.file_url" :block="false" />
+                    <InspectBufrButton :fileName="data_item.filename" :fileUrl="data_item.file_url" :block="false" />
                   </div>
                   <div class="file-actions" v-if="data_item.data">
-                    <DownloadButton :fileName="data_item.filename" :data="data_item.data" />
-                    <InspectBufrButton :fileName="data_item.filename" :data="data_item.data" />
+                    <DownloadButton :fileName="data_item.filename" :data="data_item.data" :block="false" />
```

Comment:
is block required if the default is false and if we are not reading from a variable?

Reply:
Block is not required, it defaults to false. However, since breakpoint based repeated code is often confusing to read, I felt that explicitly putting `:block="false"` makes the intention of the code clearer. It really makes no difference though.